frontend/lib/content/css.js

```diff
@@ -0,0 +1,16 @@
+const cssForExperiment = experiment => `
```

Comment:
This is probably the simplest example of a transform function. Just turns experiments into a single big CSS file.